src/lib/factory.js

```diff
@@ -25,17 +25,18 @@ class PluginFactory {
     this.metadata.prefix = opts.prefix
     this.adminPlugin = null
     this.plugins = new Map()
+    this.ready = false
```

Comment:
It would be good to include a description of the difference between "ready" and "connected" in a comment